### DISCUSSION TITLE
Welcome to SwordfishCollections Discussions!

### DISCUSSION THREAD

**Discussion by stewienj** [stewienj/SwordfishCollections#20] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GregoryHuff** [comment]
I'm not totally sure I understand how ConcurrentObservableCollection works but I'm glad it does. You saved me a lot of work today. Thanks.
